client/src/main/java/io/cloudsoft/winrm4j/client/WinRmClient.java

```diff
@@ -413,18 +409,7 @@ private synchronized WinRm createService() {
 //        JaxWsDynamicClientFactory dcf = JaxWsDynamicClientFactory.newInstance();
 //        Client client = dcf.createClient("people.wsdl", classLoader);
         winrm = service.getWinRmPort(
```

Comment:
am a tourist here but in case it helps, an alternative fix based on http://mail-archives.apache.org/mod_mbox/cxf-users/201505.mbox/%3C201505211429.t4LEN4tT026908@mx0b-00116901.pphosted.com%3E is to use `JaxWsProxyFactoryBean`

code snippet at http://learningviacode.blogspot.co.uk/2013/05/building-clients-with-cxf-api.html suggests something like

```
 JaxWsProxyFactoryBean factory = new JaxWsProxyFactoryBean();
   
 factory.setServiceClass(IEcho.class); //the service SEI
 factory.setAddress(endpointAddress);
 IEcho client = (IEcho) factory.create();
```

if something like that can work it would certainly be simpler than this and hopefully be a better solution solving the correct selection of underlying service from cxf instead of the system